src/bundle/Resources/public/scss/_modals.scss

```diff
@@ -95,3 +95,37 @@
         background-color: $ez-ground-base;
     }
 }
+
+.ez-modal--version-draft-conflict {
+    .modal-header {
+        background: $ez-color-base-pale;
+        border-bottom: none;
+
+        h5 {
```

Comment:
We should avoid styling by tag selectors. Please, implement the style update in the context of the `modal-title` class selector.